Frameworks/libmysqlclient/build-libmysqlclient.sh

```
@@ -63,18 +63,26 @@ fi
+# failed `brew --prefix` would export empty paths and the build would go on
+# with a silently broken OpenSSL.
+openssl_prefix="$(/opt/homebrew/bin/brew --prefix openssl@3)"
+if [ -z "${openssl_prefix}" ] || [ ! -d "${openssl_prefix}" ]; then
+    echo "❌ Could not resolve openssl@3 via /opt/homebrew/bin/brew (arm64). Aborting build process."
+    exit 1
+fi
+export OPENSSL_ROOT_DIR="${openssl_prefix}"
+export OPENSSL_LIB_DIR="${openssl_prefix}/lib"
+export OPENSSL_INCLUDE_DIR="${openssl_prefix}/include"
 
 /opt/homebrew/bin/cmake -S . -B $BUILD_DIR/arm64 \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)